@xen-orchestra/lite/src/components/Select.tsx

```diff
@@ -16,6 +16,7 @@ interface State {}
 
 interface Props extends SelectProps {
   additionalProps?: AdditionalProps
+  isControlled?: boolean
```

Comment:
IMO, our component should always be controlled and we should either prevent the use of `undefined` for `value` or always fallback to `''`/`[]`.